FWCore/Catalog/interface/FileLocator.h

```diff
@@ -1,22 +1,29 @@
 #ifndef FWCore_Catalog_FileLocator_h
 #define FWCore_Catalog_FileLocator_h
 
+#include "FWCore/Catalog/interface/SiteLocalConfig.h"
```

Comment:
This header is now included twice
```suggestion
```

Reply:
I removed this and will commit together with RucioCatalog turning on